ABOUT_ME.md

```diff
@@ -0,0 +1,5 @@
+INTRODUCTION:-
+
+Hello everyone, my name is Prakash Kumar.I am pursuing B.Tech from Silicon Institute of Technology.
+Currently, I am in 8th semester and doing intern at Surya Digital.My hobbies are listening songs and going to gym.
```

Comment:
```suggestion
Currently, I am in 8th semester and doing internship at Surya Digital. My hobbies are listening songs and going to gym.
```
The grammar of this sentence does not seem right to me, please use a tool like Grammarly to proofread your sentences.

Reply:
Thanks for your feedback.

Reply:
Understood.